### ISSUE TITLE
A couple of WordPress enhancements

### ISSUE THREAD

**Issue by igor-gp** [gantry/gantry5#1348]
Hey @newkind what do you think about a couple of these enhancements?

- It would be handy to have the theme options menu item show drop-downs for the tabs, so you can quickly go to the right one without having to see styles first if you don't need to

- It would be useful if Gantry built in a native `Custom` tab where people can just add their css code and it auto writes it to custom folder

**Comment by newkind** [comment]
I like the first one, although it would work only for the Base Outline, but that should be nie addition.

The second one would be redundant to the Atom functionallity.

**Comment by igor-gp** [comment]
Yeah good point regarding the atoms for css, i guess most WordPress users are used to seeing a place for custom css, and without being across the terminology they wouldn't think to go to atoms. But that's fine, part of the Gantry5 education :)

**Comment by newkind** [comment]
Hmm the only issue I have with that is since we don't create submenu pages using add_submenu_page, we need to use the urls, and because of that no matter on what theme page you'll be in - it will always show the "Styles" as the active. 

![helium theme gantry5 dev wordpress 2016-06-08 12-53-04](https://cloud.githubusercontent.com/assets/1641534/15892083/1eb6136c-2d78-11e6-8fc5-8e2caf3e1d87.jpg)

@mahagr your opinion ?

**Comment by w00fz** [comment]
I know this is WordPress but I'd like to link it to #597, would be nice to have it for both platforms, even though on joomla is a bit more complicated.

// Djamil Legato // From iPhone //

**Comment by newkind** [comment]
The thing is that in WP you can have only one theme active at a time.

**Comment by w00fz** [comment]
Yah, the approach discussed here is good for WordPress, it is different than joomla just the same idea. That's why I linked the other issue.

**Comment by mahagr** [comment]
I think I did mention (without taking a look) that this is easy to do in WP, but not so much in Joomla because of their flawed admin menu system.

I think this is a good addition to WP.

**Comment by w00fz** [comment]
So I guess the question here is how to keep the proper highlight in the sidebar for Wordpress, or should it just never be highlighted?

**Comment by newkind** [comment]
Yah, I'll have to investigate it further.
